api/ruxos_posix_api/src/imp/stdio.rs

```diff
@@ -156,6 +160,10 @@ impl super::fd_ops::FileLike for Stdout {
         Ok(self.inner.lock().write(buf)?)
     }
 
+    fn flush(&self) -> LinuxResult {
+        Ok(())
```

Comment:
I think it is necessary to check here as well. Should the `stdout` buffer be flushed here as well?